data/campaigns/Liberty/scenarios/08_Glory.cfg

```diff
@@ -169,7 +169,7 @@
             [/avoid]
         [/ai]
         [ai]
-            turns=7-FOREVER
+            turns=7-99999
```

Comment:
For reference, the problem here was that FOREVER is a WML macro, and so it would have simply needed the curly braces around it. However, since this seems to be the only place in mainline where that macro was attempted to be used, just going back to a numeric constant instead is perfectly fine.